playkit/src/main/java/com/kaltura/playkit/plugins/KalturaLiveStatsPlugin.java

```diff
@@ -235,16 +238,50 @@ private void sendLiveEvent(final long bufferTime) {
         // Parameters for the request -
         // String baseUrl, int partnerId, int eventType, int eventIndex, int bufferTime, int bitrate,
         // String startTime,  String entryId,  boolean isLive, String referrer
-        RequestBuilder requestBuilder = LiveStatsService.sendLiveStatsEvent(baseUrl, partnerId, isLive ? 1 : 2, eventIdx++, bufferTime,
-                lastReportedBitrate, sessionId, mediaConfig.getStartPosition(), entryId, isLive, PlayKitManager.CLIENT_TAG, "hls");
+        long distanceFromLive = 0;
+        if (player != null) {
+            distanceFromLive = player.getDuration() - player.getCurrentPosition();
+        }
+
+        RequestBuilder requestBuilder = LiveStatsService.sendLiveStatsEvent(pluginConfig.getBaseUrl(),
+                pluginConfig.getPartnerId(),
+                (distanceFromLive <= FIFTEEN_SEC) ? KLiveStatsEvent.LIVE.value : KLiveStatsEvent.DVR.value,
+                eventIdx++, bufferTime,
+                lastReportedBitrate,
+                sessionId, mediaConfig.getStartPosition(),
+                pluginConfig.getEntryId(),
+                isLive,
+                PlayKitManager.CLIENT_TAG,
+                (playbackProtocol != null) ? playbackProtocol : "NA",
+                pluginConfig.getContextId(),
+                Base64.encodeToString(context.getPackageName().getBytes(), NO_WRAP),
```

Comment:
@giladna make sure this format is correct by the API requirement, not sure we need to encode it here